MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -252,25 +268,37 @@ private string NormalizeFormat(string format)
                 return null;
             }
 
-            // Handle MPEG-1 container
-            if (string.Equals(format, "mpegvideo", StringComparison.OrdinalIgnoreCase))
+            var formatString = string.Empty;
+            var splitFormat = format.Split(',');
+            for (var i = 0; i < splitFormat.Length; i++)
```

Comment:
This feels overly complicated. Can't we just use string.Replace?

Reply:
I think we can, not sure why I didn't use it initially...

Reply:
As mentioned by @dmitrylyzo in chat there can be collisions if we use replace, so I opted to not change the approach.